src/main/java/org/lightcouch/CouchDbClientBase.java

```diff
@@ -195,13 +195,16 @@ public String getCookie(){
 		 * @param dbName The database name
 		 * @param confirm A confirmation string with the value: <tt>delete database</tt>
 		 */
+		@Deprecated

```

Comment:
Either remove the javadoc so that the interface's doc is used or copy the javadoc so they match